lib/inv_ca_cg.cpp

```diff
@@ -53,6 +51,153 @@ namespace quda {
     if (!param.is_preconditioner) profile.TPSTOP(QUDA_PROFILE_FREE);
   }
 
+  CACGNE::CACGNE(DiracMatrix &mat, DiracMatrix &matSloppy, SolverParam &param, TimeProfile &profile) :
+    CACG(mmdag, mmdagSloppy, param, profile), mmdag(mat.Expose()), mmdagSloppy(matSloppy.Expose()),
+    xp(nullptr), yp(nullptr), init(false) {
```

Comment:
clang-format suggested the following change:


```
-  CACGNE::CACGNE(DiracMatrix &mat, DiracMatrix &matSloppy, SolverParam &param, TimeProfile &profile) :
-    CACG(mmdag, mmdagSloppy, param, profile), mmdag(mat.Expose()), mmdagSloppy(matSloppy.Expose()),
-    xp(nullptr), yp(nullptr), init(false) {
+  CACGNE::CACGNE(DiracMatrix& mat, DiracMatrix& matSloppy, SolverParam& param, TimeProfile& profile)
+      : CACG(mmdag, mmdagSloppy, param, profile)
+      , mmdag(mat.Expose())
+      , mmdagSloppy(matSloppy.Expose())
+      , xp(nullptr)
+      , yp(nullptr)
+      , init(false)
+  {
```

<!-- identifier: clang-format -->
<!-- File: lib/inv_ca_cg.cpp -->